rusk/src/lib/vm.rs

```diff
@@ -160,7 +168,12 @@ impl Rusk {
         let provisioners = self
             .provisioners(base_commit)
             .map_err(|e| anyhow::anyhow!("Cannot get provisioners {e}"))?
-            .into_iter()
+            .filter(|(_, stake)| {
+                stake
+                    .amount
+                    .map(|(amount, _)| amount >= dusk(1000.0))
```

Comment:
There's already one for this: `MINIMUM_STAKE`

Reply:
Agree with both of you.

However the are some things that I don't like.

1. Current `MINIMUM_STAKE` is defined in `rusk-recovery` and that's not included always.
2. Moving it into `consensus` crate would need rusk-abi to properly calculate the LUX conversion
3. Even moving it to `rusk` would be a duplication, because it's defined in the stake-contract

I'll probably go for the third until we found a proper way to query the contract at startup

What do you think?

Reply:
On second thought, I could expose a new call to feed only provisioners that satisfies the contract's minimum_stake. It's the cleanest way to have a single source of truth